src/lib/components/autocompleteAddress/index.tsx

```diff
@@ -339,12 +342,13 @@ const AutocompleteAddress = ({
       {manualAddressEntry === false && (
         <div className="p-p mt8">
           {manualAddressEntryTexts?.preText || 'Or '}
-          <span
-            className="p-a fw-bold c-pointer"
+          <button
+            className={classNames(`p-a p-p fw-bold c-pointer bg-transparent`)}
```

Comment:
Just a little nitpick, we don't need the `classNames` package or the template string here, we can leave it as a normal string

Reply:
Makes perfect sense, I'll make this tiny tweak (without asking you for a new review). Thank you!